salesagility/SuiteCRM
# Display checkboxes for boolean fields in PDF templates

## Description

Replace true/false hardcoded strings by checkbox images.


true/false are hardcoded.
1- Create a checkbox field in Quotes or Invoices (for example)
2- Add that checkbox field to a PDF template
3- Use any SuiteCRM translation
4- The field will be displayed as true or false whatever the language




## Motivation and Context

- fix an issue with hardcoded strings
- more generic than just fixing this language issue (sometimes you would want to display True/False, sometimes you would want to display Yes/No)

## How To Test This

1- Create a checkbox field in Quotes or Invoices (for example)
2- Add that checkbox field to a PDF template
3- The field is displayed as checked or unchecked images

## Types of changes


### Final checklist
